code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1892,6 +1893,110 @@ GLOBAL_LIST_EMPTY(roundstart_races)
+					to_chat(humi, span_warning("[pick("Your skin feels itchy.", "You feel like burning.")]"))
+				else
+					to_chat(humi, span_warning("[pick("Your skin feels hot.", "You feel like burning.")]"))
+					humi.adjustFireLoss(gas_amount * 0.5 * delta_time)
+			if(/datum/gas/oxygen)
+				if(gas_amount > 50 && environment.temperature > 400)
+					humi.adjust_fire_stacks(gas_amount * 0.1)
+			if(/datum/gas/plasma)
+				if(isplasmaman(humi))
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.15 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.15 * delta_time)
+					continue
+				if(gas_amount > 5)
+					to_chat(humi, span_warning("[pick("Your skin stings.", "You feel like there is something oily on you.", "You feel something crawling on you!", "Your liver starts moving!", "Your right eye seems sad.", "Your left leg want to be right.")]"))
+					humi.hallucination += gas_amount * 0.5 * delta_time
+					humi.hallucination = clamp(humi.hallucination, 0, 100)
+			if(/datum/gas/pluoxium)
+				if(gas_amount > 10)
+					humi.adjustOxyLoss(- gas_amount * 0.15 * delta_time)
+			if(/datum/gas/proto_nitrate)
+				if(gas_amount > 10)
+					to_chat(humi, span_warning("[pick("Your skin burns!", "You want to remove your skin!")]"))
+					humi.apply_status_effect(/datum/status_effect/gas_rash)
+			if(/datum/gas/tritium)
+				if(gas_amount > 1 && gas_amount < 5)
+					to_chat(humi, span_warning("[pick("Your skin feels tingly.", "You feel tingly all over.")]"))
+				else
+					to_chat(humi, span_warning("[pick("Your skin burns!", "You feel weak!")]"))
+					humi.radiation += gas_amount * 0.15 * delta_time
+					var/current_stamina = humi.getStaminaLoss()
+					if(current_stamina > 25)
+						humi.adjustStaminaLoss(gas_amount * 0.15 * delta_time)
+			if(/datum/gas/water_vapor)
+				if(humi.wear_mask?.clothing_flags & BLOCK_GAS_SMOKE_EFFECT || humi.wear_mask?.clothing_flags & MASKCOVERSEYES || humi.head?.flags_cover & HEADCOVERSEYES)
+					continue
+				if(gas_amount > 5 && gas_amount < 15)
+					to_chat(humi, span_notice("You can't see beyond a few meters from you due to the fog."))
+					humi.overlay_fullscreen("tint", /atom/movable/screen/fullscreen/impaired, 1)
+				else if(gas_amount < 30)
+					to_chat(humi, span_warning("You can't see almost right next to you due to the fog."))
+					humi.overlay_fullscreen("tint", /atom/movable/screen/fullscreen/impaired, 2)
+				else
+					to_chat(humi, span_warning("The fog stops you from seeing around you!"))
+					humi.become_blind(EYES_COVERED)
+				addtimer(CALLBACK(humi, /mob/living/proc/cure_blind, EYES_COVERED), 1 SECONDS * delta_time)
+				addtimer(CALLBACK(humi, /mob/proc/clear_fullscreen, "tint", 0), 1 SECONDS * delta_time)
+			if(/datum/gas/zauker)
+				if(gas_amount > 0.5)
+					to_chat(humi, span_warning("[pick("Your skin peels off!", "You feel very nauseous!")]"))
+					humi.adjustCloneLoss(gas_amount * 0.5 * delta_time)
```

Comment:
Maybe this is just a flaw with zauker, but this effect isn't great cause cloneloss is so nasty, but it's so rare that it's almost meaningless

Reply:
yeah i think it's pretty self balanced

Reply:
you dont need the [] for that pick btw

Reply:
changed this and all the others spans to just pick() without the []